### PR TITLE
Changes to Board and Cell components

### PR DESCRIPTION
**Pull request by didibu26**
Changed indexing of Cells in the Board component and created 2 new props, row and col, so we know the coordinates of each cell. 

Updated regex expression to accept an empty string, and only numbers from 1 to 9. If input is a letter or a `number <0 or >9`, cell color changes to red.

### CODE REVIEW

File: src/app/components/board.jsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 const board = [...Array(9)]
 
 export default function Board() {
   
   const [size, setSize] = useState(9)
-  
+
   return (
-    <div className="flex flex-row">
+    <div className="flex flex-row border border-1 border-slate-500 justify-center items-center w-screen h-screen">
```
Comment (raswonders) [inline src/app/components/board.jsx:13]: you're readding border and border-1 which I've eliminated it caused weird lines in UI

File: src/app/components/cell.jsx [hunk 2]
```diff
@@ -16,6 +20,6 @@ export default function Cell( { name } ) {
     }
 
     return (
-        <input type="text" pattern="^[1-9]+$" value={number} className="w-10 h-10 border border-1 border-slate-400 text-center " onChange={(e) => handleChange(e)}></input>
```
Comment (raswonders) [inline src/app/components/cell.jsx:19]: can we have this cell inputs type number?

**Review by raswonders — CHANGES_REQUESTED** [review]
looks good, but please remove the borders and consider changin input type to number if it works

**Review by didibu26 — COMMENTED** [review]
changed in the last 2 commits

**Review by damanpr33t — APPROVED** [review]

**Review by raswonders — APPROVED** [review]
Ok looks ok, there are still some UX challenges regarding digit INPUT which we agreed - will be tackled as we go.

**Pull request merged**